docs/pages/access-controls/access-lists/guide.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: A Simple Access List Example
+description: Learn how to use Access Lists to manage and audit long lived access to Teleport managed resources.
+---
+
+<Admonition type="warning">
+Access Lists are currently in Preview mode.
+</Admonition>
+
+This guide will help you set up a basic Access List.
+
+## Prerequisites
+
+(!docs/pages/includes/commercial-prereqs-tabs.mdx!)
+
+- (!docs/pages/includes/tctl.mdx!)
+- A running Teleport cluster with some built in resources. Following the
+  [Web Application Access Guide](../../application-access/guides/connecting-apps.mdx) will create a base with a
+  simple application that we can use to demonstrate access lists.
+- An administrative user with the preset `editor` role, which will have permissions to create Access Lists.
+
+## Step 1/3. Create a test user
+
+First, we need to create a simple test user that has only the `requester` role, which has no default access
+to anything within a cluster. This user will only be used for the purposes of this guide, so you may use
+another user if you so choose.
+
+```code
+$ tctl users add test-user --roles=requester --logins=root,ubuntu,ec2-user
+```
+
+Then, follow the link provided and create credentials for this user. Try logging in to verify that no resources
```

Comment:
Will the login work if you only have requester role?

Reply:
Yes it will.